components/nimbus/src/tests/test_evaluator.rs

```diff
@@ -423,6 +423,98 @@ fn test_targeting_is_already_enrolled() {
     );
 }
 
+#[test]
+fn test_targeting_active_experiments_equivalency() {
```

Comment:
Thanks for adding the tests!! it would be great to test it end-to-end when we have the active experiments loaded into the targeting attributes but that could be a separate PR

(i.e we create a nimbus client, enroll it in an experiment, then verify that targeting by `active_experiments` behaves as expected)